test/fork/executeMarketSwapIntentGrouptest.ts

```diff
@@ -4,36 +4,36 @@ import {
   deployAccount,
   unsignedMarketSwapData,
   MarketSwapExactInput,
-  Order,
-  Strategy,
-  SignedStrategy,
+  DeclarationIntent,
+  Declaration,
+  SignedDeclaration,
   Token,
   UniV3Twap,
   UseBit,
   marketSwapExactInput_getOutput,
-  executeStrategy,
-  strategyEIP712TypedData,
-  StrategyArgs
+  executeDeclaration,
+  declarationEIP712TypedData,
+  DeclarationArgs
 } from '@brink-sdk'
 import fundWithERC20 from '../helpers/fundWithERC20'
 
-describe('executeStrategy with marketSwapExactInput', function () {
-  it('should execute a simple market swap strategy', async function () {
```

Comment:
would make more sense for the test descriptions to use "intent" here instead of "declaration"